test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -428,6 +428,36 @@ TEST_F(HystrixSinkTest, AddAndRemoveClusters) {
   // Check that old values of test_cluster2 were deleted.
   validateResults(cluster_message_map[cluster2_name_], 0, 0, 0, 0, 0, window_size_);
 }
+
+TEST_F(HystrixSinkTest, HystrixEventStreamHandler) {
+  InSequence s;
+  Buffer::OwnedImpl buffer = createClusterAndCallbacks();
+  // Register callback to sink.
+  sink_->registerConnection(&callbacks_);
+
+  // This value doesn't matter in handlerHystrixEventStream
+  absl::string_view path_and_query;
+
+  Http::HeaderMapImpl response_headers;
+  Stats::IsolatedStoreImpl listener_scope_;
+
+  NiceMock<Envoy::Server::Configuration::MockAdminStream> admin_stream_mock;
+
+  ON_CALL(admin_stream_mock, getDecoderFilterCallbacks()).WillByDefault(ReturnRef(callbacks_));
+
+  ASSERT_EQ(
+      sink_->handlerHystrixEventStream(path_and_query, response_headers, buffer, admin_stream_mock),
+      Http::Code::OK);
+
+  // Check that response_headers has been set correctly
+  EXPECT_EQ(response_headers.ContentType()->value(), "text/event-stream");
+  EXPECT_EQ(response_headers.CacheControl()->value(), "no-cache");
+  EXPECT_EQ(response_headers.Connection()->value(), "close");
+  EXPECT_EQ(response_headers.AccessControlAllowOrigin()->value(), "*");
+  EXPECT_THAT(std::string(response_headers.AccessControlAllowHeaders()->value().getStringView()),
+              testing::HasSubstr("Accept"));
```

Comment:
nit: using declarations are a good idea for `EXPECT_THAT` matchers, since without all the `testing::` it reads like a human readable language expression.